deploy/osd-storage/cluster-storage-gcp-ssd.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: ssd
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
```

We want this to be false 
```suggestion
    storageclass.kubernetes.io/is-default-class: "false"
```